docs/migration-v3.md

```diff
@@ -15,6 +15,66 @@ Consult the [announcement post](/blog/announcement-v3) to learn more about the v
 npm add --save remirror@latest @remirror/react@latest @remirror/pm@latest
 ```
 
+## ES2023 decorators
+
+Remirror v3 has been updated to use ES2023 decorators, now they have [reached Stage 3](https://github.com/tc39/proposal-decorators), and [esbuild supports them](https://github.com/evanw/esbuild/releases/tag/v0.21.0). Previously Remirror used TypeScript's experimental decorators, which use an older version of the spec.
```

Comment:
Let's don't use the term `ES2023`, which is for the stable version of ES spec published on 2023 (https://tc39.es/ecma262/2023/). 

I think `Stage 3 decorator` should be good enough. TypeScript's docs is [using] it.

Reply:
Same comment for the `ES2023` term in the JSDoc comment.